mappings/net/minecraft/entity/passive/VillagerEntity.mapping

```diff
@@ -26,22 +34,33 @@ CLASS avo net/minecraft/entity/passive/VillagerEntity
 		ARG 1 villagerData
 	METHOD a reinitializeBrain (Lvi;)V
 		ARG 1 world
+	METHOD b setGolemLastSeenTime (J)V
+		ARG 1 time
 	METHOD b setOffers (Lbgq;)V
 		ARG 1 offers
+	METHOD c hasSeenGolemRecently (J)Z
+		ARG 1 currentTime
+	METHOD d hasRecentlyWorkedAndSlept (J)Z
+		ARG 1 worldTime
 	METHOD eA getAvailableFood ()I
+	METHOD eB craftBread ()V
+	METHOD eC decayGossip ()V
 	METHOD eD spawnIronGolem ()Lard;
 	METHOD ek restock ()V
 	METHOD el playWorkSound ()V
 	METHOD en isReadyToBreed ()Z
 	METHOD eo consumeAvailableFood ()V
+	METHOD ep eatForBreeding ()V
 	METHOD eq wantsToStartBreeding ()Z
 	METHOD er canBreed ()Z
 	METHOD es hasSeedToPlant ()Z
 	METHOD et getLastRestock ()J
 	METHOD ev sayNo ()V
 	METHOD ew clearCurrentBonus ()V
+	METHOD ex lacksFood ()Z
 	METHOD ey canLevelUp ()Z
 	METHOD ez levelUp ()V
+	METHOD f getReputationFor (Lawb;)I
```

Comment:
this seems fine, but would getReputation be a more concise way of naming it?

Reply:
feared that it may be confusing because it passes down to another reputation getter in another class which has another param (only getting reputation sum from certain types of gossips)